### PR TITLE
add module sale_payment_method_auto_pay_selection

### PR DESCRIPTION
**Pull request by rami-wafaie**

### CODE REVIEW

File: sale_payment_method_auto_pay_selection/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# © initOS GmbH 2016
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{'name': 'Sale Payment Method - Selection for Automatic Payment',
+ 'version': '8.0.1.0.0',
+ 'category': '',
+ 'description': """
```
Comment (elicoidal) [inline sale_payment_method_auto_pay_selection/__openerp__.py]: remove this key (already in README)

File: sale_payment_method_auto_pay_selection/README.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+=====================================================
+Sale Payment Method - Selection for Automatic Payment
+=====================================================
+
+* Adds an selection to payment methods to configure when an automatic payment is allowed to be created.
```
Comment (StefanRijnhart) [inline sale_payment_method_auto_pay_selection/README.rst:8]: It seems this description is slightly misleading. Is this not more about being able to mark payment methods not to allow automatic payment?

**Review by susport — COMMENTED** [review]
Please can you insert data test?

**Comment by rousseldenis** [comment]
Closing this as no activity. @rami-wafaie  Please feel free to reopen it if you still need it. Take care of your PR's

**Pull request closed without merging**